datacenterlight/locale/de/LC_MESSAGES/django.po

```diff
@@ -523,35 +518,59 @@ msgstr ""
 "Deine VM ist gleich bereit. Wir senden Dir eine Bestätigungsemail, sobald Du "
 "auf sie zugreifen kannst."
 
-#~ msgid "Affordable VM hosting based in Switzerland"
-#~ msgstr "Bezahlbares VM Hosting in der Schweiz"
+#~ msgid "Enter name"
+#~ msgstr "Name"
 
-#~ msgid "Processing..."
-#~ msgstr "Abarbeitung..."
+#~ msgid "Enter email"
+#~ msgstr "E-Mail-Adresse"
 
-#~ msgid "Hold tight, we are processing your request"
-#~ msgstr "Bitte warten - wir verbeiten Deine Anfrage gerade"
+#~ msgid "Request Beta Access"
+#~ msgstr "Beantrage Beta-Zugang"
 
-#~ msgid "Some problem encountered. Please try again later."
-#~ msgstr "Ein Problem ist aufgetreten. Bitte versuche es später noch einmal."
+#~ msgid "Request Sent"
+#~ msgstr "Anfrage verschickt"
 
-#~ msgid "Submit"
-#~ msgstr "Absenden"
+#~ msgid ""
+#~ "Thank you for your subscription! You will receive a confirmation mail "
+#~ "from our team"
+#~ msgstr ""
+#~ "Vielen dank für Ihre Anmeldung. Sie erhalten in kürze eine "
+#~ "Bestätigungsmail von unserem Team"
 
-#~ msgid "Date"
-#~ msgstr "Datum"
+#~ msgid "Thank you for your request."
+#~ msgstr "Vielen Dank für Deine Anfrage."
 
-#~ msgid "Billed To:"
-#~ msgstr "Rechnungsadresse"
+#~ msgid "You are one step away from being our beta tester!"
+#~ msgstr ""
+#~ "Sie sind nur noch einen Schritt davon entfernt, unser Beta-Tester zu "
+#~ "werden!"
 
-#~ msgid "Payment Method:"
-#~ msgstr "Bezahlmethode"
+#~ msgid ""
+#~ "Currently we are running our tests to make sure everything runs perfectly."
+#~ msgstr ""
+#~ "Momentan testen wir die Beta-Umgebung um sie für Ihren Gebrauch "
+#~ "sicherzustellen."
 
```

Comment:
"Momentan testen wir die Beta-Umgebung um sie für Ihren Gebrauch sicherzustellen."
->Let's lift the DE part of this translation, and leave only EN. Reason is that it doesn't really makes sense in DE and I'm not able to find a good translation of EN(which makes sense actually) for the moment. 
Since we're not using this Beta phrases now, we can put new lines in the future when we need.